### PR TITLE
Reserve core saving error

### PR DESCRIPTION
**Pull request by favadi**

### CODE REVIEW

**Comment by favadi** [comment]
[Saving records in reservecore.Trade/Widthraw failed](https://trello.com/c/ZTXdOHGl/86-saving-records-in-reservecoretrade-widthraw-failed)

**Comment by favadi** [comment]
[Unhandled errors](https://trello.com/c/hBcWsYVl/32-unhandled-errors)

File: core/reserve_core.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func (self ReserveCore) Trade(
 		"",
 		timepoint,
 	)
-	if err != nil {
+	if sErr != nil {
 		log.Printf("Error save activity: %s", err.Error())
```
Comment (cyberliem) [inline core/reserve_core.go]: should be sErr instead of err

File: core/reserve_core.go [hunk 2]
```diff
@@ -235,6 +235,11 @@ func (self ReserveCore) Withdraw(
 		"",
 		timepoint,
 	)
+
+	if sErr != nil {
+		log.Printf("failed to save withdraw record: %s", err.Error())
```
Comment (cyberliem) [inline core/reserve_core.go]: Same here.

File: core/reserve_core.go (hunk 1 above)
Comment (favadi) [reply core/reserve_core.go]: fixed in 1afb0c62a08d3561b8e079d6298efc57e81cbe62

**Pull request merged**